README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/pdxcapstone/API-Test-Automation.svg?branch=master)](https://travis-ci.org/pdxcapstone/API-Test-Automation)
+[![Build Status](https://travis-ci.org/pdxcapstone/API-Test-Automation.svg?branch=sails_test)](https://travis-ci.org/pdxcapstone/API-Test-Automation)
```

Comment:
Are you sure the branch should change to sails_test?  I think we would still want the CI to report on the master status.